lib/dialogs/about.jsx

```diff
@@ -110,7 +112,7 @@ export default React.createClass({
               href="https://automattic.com/"
               rel="noopener noreferrer"
             >
-              &copy; 2016 Automattic, Inc.
+              &copy; {thisYear} Automattic, Inc.
```

Comment:
also available without any external libraries as `new Date().getFullYear()` (already in local time that one)

Reply:
WFM!